media-source-respec.html

```diff
@@ -946,24 +920,41 @@
       <h2>SourceBuffer Object</h2>
 
 
-      <dl title="enum AppendMode" class="idl">
-        <dt>segments</dt>
-        <dd>
+      <div><pre class="idl">enum AppendMode {
+    "segments",
+    "sequence"
+};</pre><table class="simple" data-dfn-for="AppendMode" data-link-for="AppendMode"><tbody><tr><th colspan="2">Enumeration description</th></tr><tr><td><dfn><code id="idl-def-AppendMode.segments">segments</code></dfn></td><td>
           <p>The timestamps in the media segment determine where the <a def-id="coded-frames"></a> are placed in the presentation. Media segments can be appended in any order.</p>
-        </dd>
-        <dt>sequence</dt>
-        <dd>
+        </td></tr><tr><td><dfn><code id="idl-def-AppendMode.sequence">sequence</code></dfn></td><td>
           <p>Media segments will be treated as adjacent in time independent of the timestamps in the media segment. Coded frames in a new media segment will be placed immediately after the coded
             frames in the previous media segment. The <a def-id="timestampOffset"></a> attribute will be updated if a new offset is needed to make the new media segments adjacent to the previous media segment.
             Setting the <a def-id="timestampOffset"></a> attribute in <a def-id="AppendMode-sequence"></a> mode allows a media segment to be placed at a specific position in the timeline without any knowledge
             of the timestamps in the media segment.
           </p>
-        </dd>
-      </dl>
-
-      <dl title="interface SourceBuffer : EventTarget" class="idl">
-        <dt>attribute AppendMode mode</dt>
-        <dd>
+        </td></tr></tbody></table></div>
+
+      <div><pre class="idl">interface SourceBuffer : EventTarget {
```

Comment:
(Ditto: boolean, TimeRanges, double, {Audio,Video,Text}TrackList, unrestricted double, EventHandler, void, link to appendBuffer(...) detail, ArrayBuffer, ArrayBufferView, ReadableStream, unsigned long long are all missing links versus the previous index.html in at least the IDL section. Likewise, the attribute and method details probably have similar problems.)